salt/states/consul.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+'''
+Consul Management
+=================
+
+The consul module is used to create and manage Consul ACLs
+
+.. code-block:: yaml
+
+    acl_present:
+      consul.acl_present:
+        - id: 38AC8470-4A83-4140-8DFD-F924CD32917F
+        - name: acl_name
+        - rules: node "" {policy = "write"} service "" {policy = "read"} key "_rexec" {policy = "write"}
+        - type: client
+        - consul_url: http://localhost:8500
+
+    acl_delete:
+       consul.acl_absent:
+         - id: 38AC8470-4A83-4140-8DFD-F924CD32917F
+'''
+
+
+def _acl_changes(name, id=None, type=None, rules=None, consul_url=None, token=None):
+    '''
+       return True if the acl need to be update, False if it doesn't need to be update
+    '''
+    info = __salt__['consul.acl_info'](id=id, token=token, consul_url=consul_url)
+
+    if info['res'] and info['data'][0]['Name'] != name:
+        return True
+    elif info['res'] and info['data'][0]['Rules'] != rules:
+        return True
+    elif info['res'] and info['data'][0]['Type'] != type:
+        return True
+    else:
+        return False
+
+
+def _acl_exists(name=None, id=None, token=None, consul_url=None):
+    '''
+       If acl exists, return True. Otherwise, return False.
```

Comment:
This seems misleading. This makes it seem like the function returns a boolean when it actually returns a dictionary.